File: Raju Contribution/MATLAB Files/findCV.m
function cvVal = findCV (resultant)
% This function calculates the coefficient of variation (CV)
% INPUT resultant -> input vector
% OUTPUT cvVal -> scalar, coefficient of variation value

% Finding MEAN
meanVal = mean(resultant);%#codegen

% Finding standard deviation
varVal = std(resultant(:));%standard deviation

% Finding Coeeficient of variation
cvVal=varVal/meanVal;%test_obtData

File: C_New Coding/find_CV.c
double find_CV(int *resultant,int rows){
	double temp_mean=0,mean,sdev,cvVal,temp_var=0;
	int i;
	//Finding mean first
	for(i=0;i<rows;i++){  		
  			temp_mean+=*(resultant+i); 			  
	}
	mean=temp_mean/rows;
	
	//Finding Standard Deviation
	for(i=0;i<rows;i++){
		temp_var+=(*(resultant+i)-mean)*(*(resultant+i)-mean);
	}
	sdev=sqrt(temp_var/(rows));
	
	//Finding Coefficient of variation
	cvVal=mean/sdev;
	
	return cvVal;	
}
